src/app/app.component.html

```diff
@@ -341,7 +347,7 @@
       </g>
     </svg>
 
-    <span>{{ title }} app is running!</span>
+    <span>{{ title }} app is runningAAAAAAAAAAA!</span>
```

Comment:
necesito que cambie el AAA
```suggestion
    <span>{{ title }} app is runningAA!</span>
```